client/components/tinymce/plugins/wpcom-view/views/embed/index.js

```diff
@@ -46,11 +41,11 @@ export default class EmbedViewManager extends EventEmitter {
 		}
 	}
 
-	addListener( event, listener ) {
+	addListener( event, listener, store ) {
 		super.addListener( event, listener );
-
 		if ( 'change' === event && 1 === this.listeners( event ).length ) {
-			sites.addListener( event, this.sitesListener );
+			this.store = store;
+			this.unsubscribeStore = store.subscribe( this.sitesListener );
```

Comment:
Unfortunately this one will fire on any store change, and we should avoid this if possible. I recall this one being tricky.

Perhaps we could do something here using the [lib/middleware](https://github.com/Automattic/wp-calypso/blob/master/client/state/lib/middleware.js#L54)?